invenio_drafts_resources/resources/records/resource.py

```diff
@@ -28,3 +28,16 @@ def create(self):
         item = self.service.create(
             g.identity, data, links_config=self.config.draft_links_config)
         return item.to_dict(), 201
+
+    def search(self):
+        """Perform a search over the items."""
+        identity = g.identity
+        status = resource_requestctx.url_args.pop("status", ["published"])
```

Comment:
Yes, because the url args parser puts them in an array, so when passed it would be `['drafts']` or `['published']` so to avoid array vs string distinction.